build.sbt

```diff
@@ -775,6 +779,52 @@ lazy val zioJson: ProjectMatrix = (projectMatrix in file("json/zio"))
   )
   .dependsOn(core)
 
+//grpc
+lazy val protobuf: ProjectMatrix = (projectMatrix in file("grpc/protobuf"))
```

Comment:
maybe this module needs a better name - looking here I don't know what it does :)

Reply:
hm I guess it generates .proto files? are these proto files compatible with that pbdirect generates? I see it has some annotations there

Reply:
Yes, this module is meant to generate a proto file from endpoints definitions. The generated file is compatible with pbdirect codecs